build/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM busybox
+FROM buildpack-deps:jessie-curl
```

Comment:
I wasn't picky. This is what the `golang` image relies on internally, so it seemed like a good choice for now.